packages/abstract-core/src/objects/fee.rs

```diff
@@ -71,6 +71,76 @@ impl Fee {
     }
 }
 
+/// A wrapper around Coin to help handle fixed fees (with multiples).
+#[cosmwasm_schema::cw_serde]
+pub struct FixedFee {
+    /// Fee to be paid for a unit of a service
+    fee: Coin,
+}
+
+impl FixedFee {
+    pub fn new(fee: Coin) -> Self {
+        FixedFee { fee }
+    }
+    /// Allows to collect the fee multiple times
+    /// E.g., for namespaces, you want to charge the number of claimed namespaces times the fee for 1 namespace
+    pub fn quantity(&self, qty: u128) -> Self {
```

Comment:
this can take `self`, reduces the cloning. 

In general if a function can omit a clone by taking an owned value it should do that.

Reply:
Yes, corrected